upnpdiscovery/service.go

```diff
@@ -0,0 +1,229 @@
+// Package upnpdiscovery provides the discovery service that wraps UPnP integration for the viamrtsp module
+package upnpdiscovery
```

Comment:
Can you make a cmd/upnpdiscovery script to run, that has been useful for onvif discovery on machine.